requirements.txt

```diff
@@ -8,7 +8,7 @@ g2p>=0.5.20210825
 lxml==4.6.5
 networkx==2.5
 numpy>=1.16.4
-soundswallower==0.1.5
+soundswallower>=0.2.2
```

Comment:
I'm surprised you didn't use `~=0.2.2`, but this is fine too.

Reply:
Hmm in fact yes it should be ~= 0.2.2 now that I think of it...